GameReadyValidator/src/test/java/pl/przemyslawjakubowski/tajgertim/gamereadyvalidator/gamevalidation/ConfirmationsReceivedEventListenerTest.java

```diff
@@ -0,0 +1,81 @@
+package pl.przemyslawjakubowski.tajgertim.gamereadyvalidator.gamevalidation;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pl.jakubowskiprzemyslaw.tajgertim.models.confirmation.*;
+import pl.jakubowskiprzemyslaw.tajgertim.queues.Queues;
+import pl.jakubowskiprzemyslaw.tajgertim.services.QueueService;
+
+import static org.mockito.Mockito.*;
+
+@Test
+public class ConfirmationsReceivedEventListenerTest {
+
+    private ConfirmationReceivedEventListener confirmationReceivedEventListener;
+    private QueueService queueService;
+    private Confirmations confirmationsMock;
```

Comment:
Why can't you use real confirmations?

Reply:
We are using real confirmations also, but in places where we need to verify that specific method was called, this one is needed

Reply:
verify(confirmationsMock).addConfirmation(confirmation); 
instead you can check the state of the Confirmations object and check your confirmation is present

Reply:
done